twistlock/datadog_checks/twistlock/twistlock.py

```diff
@@ -14,6 +14,11 @@
 from .config import Config
 from .utils import normalize_api_data_inplace
 
+try:
+    import orjson as json
+except ImportError:
+    import json
+
```

Comment:
Can we create a new util in the base package that does just that?

```python
from datadog_checks.base.utils.serialization import json
```

```python
# serialization.py

try:
    import orjson as json
except ImportError:
    import json

__all__ = ["json"]
```

Reply:
Was gonna do a separate PR, but yeah might as well do it here